R/fn_analysis.R

```diff
@@ -1,22 +1,39 @@
+#' Converts long data to wide if necessary, then applies freq_wrap().
+#' 
+#' @param data Input dataset.
+#' @param metaoutcome "Continuous" or "Binary".
+#' @param treatment_list Data frame containing the treatment ID ('Number') and the treatment name ('Label').
+#' @param outcome_measure "MD", "SMD", "OR", "RR", or "RD".
+#' @param modelranfix "fixed" or "random".
+#' @param excluded = Vector of studies to be excluded for sensitivity analysis.
+#' @return See output from freq_wrap().
 frequentist <- function(data, metaoutcome, treatment_list, outcome_measure, modelranfix, excluded=c()){
-  data_wide <-  entry.df(data, metaoutcome) # Transform data to wide form
+  data_wide <-  entry.df(data = data, CONBI = metaoutcome) # Transform data to wide form
   
   # Subset of data when studies excluded
   if (length(excluded) > 0) {
     data_wide <- dplyr::filter(data_wide, !Study %in% excluded)
   }
   
   # Use the self-defined function, freq_wrap
-  return(freq_wrap(data_wide, treatment_list, modelranfix, outcome_measure, metaoutcome, 
-                   ref_alter(data, metaoutcome, excluded, treatment_list)$ref_sub))
+  return(freq_wrap(data = data_wide, treat_list = treatment_list, model = modelranfix,
+                   outcome = outcome_measure, CONBI = metaoutcome,
+                   ref = ref_alter(data = data, metaoutcome = metaoutcome, excluded = excluded,
```

Comment:
Can you move this call to `ref_alter()` outside of the call to `freq_wrap()` please? It'd make it  bit clearer what's going on